core/src/main/java/org/apache/accumulo/core/client/admin/TableOperations.java

```diff
@@ -808,4 +812,64 @@ void setSamplerConfiguration(String tableName, SamplerConfiguration samplerConfi
    * @since 1.8.0
    */
   SamplerConfiguration getSamplerConfiguration(String tableName) throws TableNotFoundException, AccumuloException, AccumuloSecurityException;
+
+  /**
+   * Entry point for retrieving summaries with optional restrictions.
+   *
+   * <p>
+   * In order to retrieve Summaries, the Accumulo user making the request will need the {@link TablePermission#GET_SUMMARIES} table permission.
+   *
+   * <p>
+   * Accumulo stores summary data with each file in each tablet. In order to make retrieving it faster there is a per tablet server cache of summary data. The
+   * size of this cache is determined by the property {code tserver.cache.summary.size}. When summary data for a file is not present, it will be retrieved using
+   * threads on the tserver. The property {@code tserver.summary.retrieval.threads} determines the max number of threads the tserver will use for this.
+   *
+   * <p>
+   * Since summary data is cached, its important to use the summary selection options to only read the needed data into the cache.
+   *
+   * <p>
+   * Summary data will be merged on the tablet servers and then in this client process. Therefore it's important that the required summarizers are on the
+   * clients classpath.
+   *
+   * @since 2.0.0
+   * @see Summarizer
+   */
+  SummaryRetriever getSummaries(String tableName) throws TableNotFoundException, AccumuloException, AccumuloSecurityException;
```

Comment:
I renamed it to `summarize()`  still not sure about that name, but I like it better than `getSummaries()`